gruntfile.js

```diff
@@ -236,6 +231,17 @@ module.exports = function(grunt) {
                         replacement: 'API_KEY = \'SmMCEZxMRoNizToppows\''
                     }]
                 }
+            },
+            'gh-pages': {
+                files: {
+                    'public/app.json': 'public/app.json'
+                },
+                options: {
+                    replacements: [{
+                        pattern: /\/develop\//g,
+                        replacement: '/' + buildTarget + '/'
```

Comment:
Ideally the pattern should be more flexible than this (I'll look at doing some path-based-fancy-regex)